package.json

```diff
@@ -7,6 +7,7 @@
     "semver": "^5.3.0"
   },
   "devDependencies": {
+    "child_process": "^1.0.2",
```

Comment:
Why? `child_process` is of the node core modules: https://nodejs.org/api/child_process.html